source/dotnet/global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.0.100-preview6-012264"
```

Comment:
>3.0.100-preview6-012264" [](start = 16, length = 24)

is this still supposed to be preview?

Reply:
The package that was just released is 3.0.100 (aka v3.0.0), if we want to build against it we must wait for the build agents to support it as the MSBuild they have is 16.2 and 3.0.100 requires 16.3 
https://dotnet.microsoft.com/download/dotnet-core/3.0

preview 6 is the latest version for the SDK we can use as of this time

Reply:
This is the error I got when trying to use 3.0.100
Error: Version 3.0.100 of the .NET Core SDK requires at least version 16.3.0 of MSBuild. The current available version of MSBuild is 16.2.37902.0.

Reply:
ok thanks -- can we loop in the ES folks + Fabian/Daniel from WPF team and make sure we can get an ETA on when RTM release would be supported on the build agents? @paulcam206 FYI.

---
In reply to: [328866285](https://github.com/microsoft/AdaptiveCards/pull/3156#discussion_r328866285) [](ancestors = 328866285)